src/main/cluster-manager.ts

```diff
@@ -50,23 +42,8 @@ export class ClusterManager {
   }
 
   getClusterForRequest(req: http.IncomingMessage): Cluster {
-    let cluster: Cluster = null
-
-    // lens-server is connecting to 127.0.0.1:<port>/<uid>
-    if (req.headers.host.startsWith("127.0.0.1")) {
-      const clusterId = req.url.split("/")[1]
-      if (clusterId) {
-        cluster = this.getCluster(clusterId)
-        if (cluster) {
-          // we need to swap path prefix so that request is proxied to kube api
-          req.url = req.url.replace(`/${clusterId}`, apiKubePrefix)
-        }
-      }
-    } else {
-      const id = req.headers.host.split(".")[0]
-      cluster = this.getCluster(id)
-    }
-
-    return cluster;
+    logger.debug(`getClusterForRequest(): ${req.headers.host}${req.url}`)
+    const clusterId = req.headers.host.split(".")[0]
+    return this.getCluster(clusterId)
```

Comment:
Why logic has been removed? Don't we use `127.0.0.1:<port>/uuid` anymore anywhere?

Reply:
Seems so.. Looks like outdated code from very beginning of lens-app.

Reply:
I asked because I `port/uuid` addresses were used when doing requests outside of renderer (wildcard dns to localhost works only within browser scope). Anyway I don't like to see changes that are totally outside of PR scope, it makes review unnecessary complicated.

Reply:
Ok, reverted back. Someone should check if this is actually works, top comment was saying:
`// lens-server is connecting to 127.0.0.1:<port>/<uid>`
which unclear in terms of what you saying how this might be used outside..